promql/promqltest/testdata/subquery.test

```diff
@@ -102,15 +102,15 @@ eval instant at 80s rate(metric[1m])
 
 # No extrapolation, [2@20, 144@80]: (144 - 2) / 60
 eval instant at 80s rate(metric[1m:10s])
-  {} 2.366666667
+  {} 2.4
 
 # Only one value between 10s and 20s, 2@14
 eval instant at 20s min_over_time(metric[10s])
   {} 2
 
-# min(1@10, 2@20)
+# min(2@20)
 eval instant at 20s min_over_time(metric[10s:10s])
-  {} 1
+  {} 2
```

Comment:
The tests intends to touch two samples, so the range has to be changed. I propose `min_over_time(metric[15s:10s])`.